src/Behat/MinkExtension/ServiceContainer/Driver/ZombieFactory.php

```diff
@@ -44,6 +44,10 @@ public function configure(ArrayNodeDefinition $builder)
                 ->scalarNode('server_path')->defaultNull()->end()
                 ->scalarNode('threshold')->defaultValue(2000000)->end()
                 ->scalarNode('node_modules_path')->defaultValue('')->end()
+                ->arrayNode('options')
```

Comment:
please add `useAttributeAsKey('name')`. The value of the argument is not really useful (as we don't support XML config files, and YAML can define maps natively), but the call itself is necessary to ensure that keys are not lost when merging 2 configs together (and Behat relies on merging config between the default profile and other profiles for insrtance). Without this call, merging treats the array as a sequence, and so reindexes everything with numeric keys.

Reply:
@stof has @berliner changed what you've requested? If so, then this PR can be merged.

Reply:
I can see in the code that it has been change.
`->useAttributeAsKey('name')` is on line 48

Reply:
I see. Looks good then.